accel-pppd/redis/redis.c

```diff
@@ -316,6 +316,16 @@ static void* ap_redis_thread(void* arg)
 			continue;
 		}
 
+		uint64_t num = 0;
+		ssize_t bytes_read;
+
+		if ((bytes_read = read(epev[0].data.fd, &num, sizeof(uint64_t))) < 0){
+			log_error("ap_redis: Could not read epoll fd: %d (%s)\n", errno, strerror(errno));
+			continue;
+		}
+
+		log_debug("ap_redis: Epoll fd content: %lu.\n", num);
```

Comment:
This message is not really needed to fix the issue.

```suggestion

```

Reply:
It's debug output useful to diagnose multifarious issues in our epoll event handling.

Reply:
Feel free to remove the line if it bugs you.